src/index.html

```diff
@@ -6,13 +6,67 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="Apple computer"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__subtitle">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-label">Price:</div>
```

Comment:
`<span>` element is commonly used for inline text formatting and is not a semantic tag, much like `<div>` is used for block-level elements. While <span> does not carry any semantic meaning by itself, it can still be beneficial for code readability and organization when used appropriately.